Autopilot/CMakeLists.txt

```diff
@@ -194,4 +194,69 @@ elseif(${KIND_OF_BUILD} STREQUAL "UNIT_TESTS")
 
 #########
 
+
+elseif(${KIND_OF_BUILD} STREQUAL "SIMULATION")
+
+  add_definitions(-DSIMULATION)
+
+  set(SIM_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../Simulink-Sim/Infinion)
+
+  # allow debugging if required
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
+
+  set(CMAKE_CXX_STANDARD 11)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+
+  include_directories(
+    ${CMAKE_CURRENT_SOURCE_DIR}/../Common/Inc
+    ${CMAKE_CURRENT_SOURCE_DIR}/Inc/
+    ${CMAKE_CURRENT_SOURCE_DIR}/AttitudeManager
```

Comment:
Do we not need to include the PathManager directory when building for simulation?

Reply:
Yes! As soon as the path manager state machine is built, the Simulation will run all of autoPilot as if it was a black box.